multimodal/tests/unittests/test_object_detection.py

```diff
@@ -35,7 +35,7 @@ def test_mmdet_object_detection_inference(checkpoint_name):
         pipeline="object_detection",
     )
 
-    pred = predictor.predict({"image": [mmdet_image_name] * 10})  # test batch inference
+    pred = predictor.predict({"image": [mmdet_image_name] * 10}, realtime=False)  # test batch inference
```

Comment:
What will happen if `realtime=True`?

Reply:
Two errors are observed:

1. The `samples_per_gpu` is kind of hard-coded into the collate function of the ImageProcessor. So, it can't handle the dynamic batch size in realtime inference.
2. The current precision control can cause output inconsistencies for the ORC recognition. The precision context manager requires improvement.

Reply:
Thanks. We may fix it in another PR.

Reply:
The problem is that this PR will make the prediction of mmdet / mmocr unusable. We may just set `realtime=False` by default in `.predict`, and turn on `realtime=None` once we have fixed the problem in MMOCR / MMDet.

Reply:
Good point! Will fix this ASAP.